service/controllers/messages.go

```diff
@@ -98,66 +98,104 @@ func routes(routeObject response, w http.ResponseWriter) {
+	// } else if len(messageArr)==2 {
+	// 	matchPars=messageArr[0]
+	// 	secondPars = messageArr[1]
+	// } else if(len(messageArr)==1) {
+	// 		matchPars=messageArr[0]
+	// }
+
+	// var remainingString string
+	// var strArr []string
+	// var strArr0 string
+	// var strArrDiff []string
+	// strArrDiff = strings.Split(message, " ")
+	// if strings.ToLower(matchPars) == "google" || strings.ToLower(matchPars) =="yahoo" || strings.ToLower(matchPars) =="bing" || strings.ToLower(matchPars) =="search" || strings.ToLower(matchPars) =="youtube"|| strings.ToLower(matchPars) == "watch"||strings.ToLower(matchPars) =="videos" || strings.ToLower(matchPars) =="images" || strings.ToLower(matchPars) =="image" || strings.ToLower(matchPars) =="meaning"{
+	// 	if strings.ToLower(secondPars) == "search" || strings.ToLower(secondPars) =="google" ||strings.ToLower(secondPars) =="yahoo" ||strings.ToLower(secondPars) =="bing" ||strings.ToLower(secondPars) =="videos" ||strings.ToLower(secondPars) =="youtube"||strings.ToLower(secondPars) =="for" || strings.ToLower(secondPars) =="of"{
+	// 		if strings.ToLower(thirdPars) == "for" || strings.ToLower(thirdPars) =="videos" {
+	// 			strArr0=matchPars+" "+secondPars+" "+thirdPars
+	// 			strArr=strings.Split(strArr0," ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		} else {
+	// 			strArr0=matchPars+" "+secondPars
+	// 			strArr=strings.Split(strArr0," ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		}
+	// 	} else {
+	// 			strArr = strings.Split(matchPars, " ")
+	// 			strArrDiff = strings.Split(message, " ")
+	// 			remainingString=strings.Join(stringDifference(strArr,strArrDiff)," ")
+	// 			fmt.Println(remainingString)
+	// 		}
+	// }
+
+	message := routeObject.message
+	messageArr := strings.Fields(message)
+	var a []string
+	priority := []string{"images", "image", "video", "videos", "watch", "youtube", "symptoms", "medicine", "weather", 
+	"meaning", "google", "yahoo", "bing", "search"}
+	for i := 0; i < len(messageArr); i++ {
+		for j := 0; j < len(priority); j++ {
+			if (messageArr[i] == priority[j]) {
+				a = append(a, messageArr[i])
+				if i < len(messageArr) {
+					messageArr = append(messageArr[:i], messageArr[i+1:]...)
+					i = i - 1
+				} else {
+					fmt.Println("Position Invalid")
+				}
+				break;
 			}
+		}
 	}
+	matchPars := ""
+	remainingString := ""
+	if len(a) > 0 {
+		sort := customSort(a, priority, len(a), len(priority))
+		matchPars = sort[0]
+		remainingString = strings.Join(messageArr[:]," ")
+		messageArr = append([]string{matchPars}, messageArr...)
+		fmt.Println("Remaining String:", remainingString)
+		fmt.Println("matchPars: ", matchPars)
+	} else {
+		remainingString = strings.Join(messageArr[:], " ")
+		fmt.Println("messageArr:", messageArr)
+		fmt.Println("Remaining String:", remainingString)
+		fmt.Println("matchPars: ", matchPars)
```

Comment:
please remove ***print*** statements if the code is working according to your plan